change/@microsoft-teams-js-e23ac78a-fff1-4ea1-919e-9b9e06ad3af7.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "gitmagic additions to 2.0-preview for February 28, 2022",
```

Comment:
For all git magic PR changefiles, let's describe the changes in the source code that's being brought in on top of explaining that these are changes being brought in from master. (Another thing I'll be adding to our wiki- I think it would be good to start each git magic changefile comment with the same git magic 'title'. today we can start by just appending `(Changes merged in from v1)` to the beginning of the changefile comment. We can discuss what this should be in the retro too.)

Reply:
That's a great suggestion! I'll append that to the comment :)